tests/utils/test_utils.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2019-2024 Intel Corporation
```

Comment:
@zhiltsov-max , what is the policy regarding Copyright inside Datumaro?

Reply:
I copypasted the code, as well as the header, from the upstream: https://github.com/openvinotoolkit/datumaro/blob/develop/tests/utils/test_utils.py#L467
I am not sure it will be right to change the header.